bin/ensure-node-modules-present.js

```diff
@@ -0,0 +1,13 @@
+const spawnSync = require( 'child_process' ).spawnSync;
+const fs = require( 'fs' );
+
+if ( ! fs.existsSync( 'node_modules' ) ) {
```

Comment:
> My previous comment was a bit sparse. What I meant was: I understand that this is a quicker way to run the linting process, but it will only be useful if one changes branches and immediately tries to commit something without previously check that it works (distclean, doing an install by starting, etc). Is this the use case we are trying to address?

That was my thought at first, but turns out there are several valid workflows that don't end up doing `npm run install-if-needed`: testing your changes in Docker, in a VM, just running the tests...